test_conformance/api/test_api_min_max.cpp

```diff
@@ -19,6 +19,10 @@
 #include <ctype.h>
 #include <string.h>
 
+// Minimum limit for OpFunctionCall actual arguments is 255 according to
+// https://www.khronos.org/registry/SPIR-V/specs/unified1/SPIRV.html#_universal_validation_rules
```

Comment:
```suggestion
// https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#_universal_limits
```

AFAICT it's actually the limits not the validation rules section that says 255.